blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/AstyanaxReader.java

```diff
@@ -326,9 +326,14 @@ public static String getType(Locator locator) {
 
     public MetricData getDatapointsForRange(Locator locator, Range range, Granularity gran) {
         try {
+            //TODO: If we stop processing string metrics, we can get rid of this and always return numeric
+            //Questions: Do we care about pre-agg types
```

Comment:
Has this question been answered? IMO we will be storing rollup types for pre-agg rollup types, so rollup type will never be `null`?